job_manager/job_manager.py

```diff
@@ -30,43 +30,42 @@ def _get_job_hash(self, func, arglist):
         # use the called function's name
         hashfunc.update(bytes(func.__name__, "utf-8"))
         # and the repo list we're passing to it
-        hashfunc.update(bytes(str(frozenset(arglist)), "utf-8"))
+        hashfunc.update(bytes(str(repo), "utf-8"))
         # grab the hex hash that's been generated.
         h = hashfunc.hexdigest()
 
         return h
 
     # dbmc is "database manager config"
-    def add_job(self, func, dbmc, repolist):
+    def add_job(self, func, dbmc, repo):
 
         # get a hash of the function used and the args supplied
-        job_hash = self._get_job_hash(func, repolist)
+        job_hash = self._get_job_hash(func, repo)
 
-        # add a job to our queue, 10 minute timeout (6000ms), id of it's hash.
-        job = self._rq.enqueue(func, dbmc, repolist, job_id=job_hash, job_timeout=6000)
+        # add a job to our queue, 1 day timeout (86400 sec), id of its hash, retry if failed
+        job = self._rq.enqueue(func, dbmc, repo, job_id=job_hash, result_ttl=86400, retry=Retry(max=3, interval=[5, 10, 15]))
 
         return job_hash
 
-    def get_job(self, func, arglist):
+    def get_job(self, func, repo):
 
         # get an identifying hash of the function used and the args supplied
-        job_hash = self._get_job_hash(func, arglist)
+        job_hash = self._get_job_hash(func, repo)
 
         # grab the job from the job queue
         job = self._rq.fetch_job(job_hash)
 
         # check if job was fetched from queue
         if job is not None:
-            logging.debug(f"CACHE HIT: {func.__name__}")
```

Comment:
Why take this out?

Reply:
Could leave it in if preferred, it was just polluting the logs because after the first time the job is enqueued, this fires for all of the callbacks, e.g. 5x each 1.2 seconds, which was excessive.